templates/engage/14-04-esm.html

```diff
@@ -10,7 +10,18 @@
 
 <section class="p-strip--image is-deep js-takeover is-dark p-takeover--compliance">
   <div class="u-fixed-width navigation-logo-engage">
-    <a href="/"><img src="https://assets.ubuntu.com/v1/f263d9c4-logo-ubuntu-white.svg" width="143" height="auto" alt="Ubuntu"></a>
+    <a href="/">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/f263d9c4-logo-ubuntu-white.svg",
+          alt="Ubuntu",
+          height="32",
+          width="143",
+          hi_def=True,
+          loading="auto"
+        ) | safe
+      }}
+    </a>
   </div>
   <div class="row u-vertically-center">
```

Comment:
can you remove this here?
```suggestion
  <div class="row">
```